### DISCUSSION TITLE
Command execution time very long

### DISCUSSION THREAD

**Discussion by stychu** [cypress-io/cypress#14531] [Questions and Help]
Why the command execution time takes almost double the time of the actual tests? 
What is happening here?

Can I reduce somehow this timing?? I can't convince my coworkers if it takes so long to use cypress.

Tests execute in 1:16 whereas time that takes from the command (process) execution to finishing it takes 1:53...  Is there something that I can do about it ?

<img width="1475" alt="Screenshot 2021-01-13 at 16 40 40" src="https://user-images.githubusercontent.com/9383657/104474425-48e44a00-55be-11eb-87c4-22ef1e51ea1e.png">

**Comment by bahmutov** [comment]
Cypress Electron app takes time to start. Then during the run Cypress opens and closes a browser instance to run every spec. There is also a video encoding time after the spec.

**Reply by stychu** [reply]
can I instead enforce it doesnt close and reopen instance every run spec?

**Comment by bahmutov** [comment]
Unfortunately no. We do it to isolate the specs from each other.

**Comment by stychu** [comment]
Oh this is unfortunate. Will there be a configuration option to not restart the browser for each test spec in the future? Or it's not likely and you don't want to deal with possible bugs because of that?

**Reply by bahmutov** [reply]
No, we think isolating the browser for each spec file brings many advantages and makes tests less likely to depend on each other. I just want to add that if running test takes too long on a single machine, you can find ways to speed them up https://www.cypress.io/blog/2019/10/29/split-a-very-long-cypress-test-into-shorter-ones-using-app-actions/ or to run tests on CI in parallel https://on.cypress.io/parallelization
